tests/cogs/test_hiring_timeline.py

```diff
@@ -0,0 +1,30 @@
+from unittest.mock import AsyncMock, MagicMock
+
+import pytest
+
+from otter_welcome_buddy.cogs import hiring_timelines
+
+
+@pytest.mark.asyncio
+async def test_register_cog(mock_bot):
+    # Arrange
+    mock_bot.add_cog = AsyncMock()
```

Comment:
If you'll be doing this in all the tests, u can add it to the `conftest`.

Reply:
It loses the idea of letting the test talk by itself, in some occasions it is not necessary to call .add_cog